src/main/java/duke/main/Parser.java

```diff
@@ -67,7 +67,7 @@ private static Date parseDate(String date) throws DukeException {
         }
     }
 
-    private static boolean isDate(String input) {
+    static boolean isDate(String input) {
```

Comment:
Great code all around! :) Maybe you could consider the edge cases for date, for example, if the user enter something like 45/52/3012 1800 by either deadline or event.